.github/workflows/stats_lambda.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow executes a Lamdba function that recording metrics for each successfully merged PR.
+#
+# You can adjust the behavior by modifying this file.
+# For more information, see:
+# https://github.com/marketplace/actions/invoke-aws-lambda
+
+name: Invoke runTheNumbers lambda functions
+on:
+   pull_request:
+       types:
+       - closed
+   workflow_dispatch:
+env:
+   COMMIT: ${{github.event.pull_request.head.sha}}
+jobs:
+  if_merged:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
Each of these steps should start with a name field so they're easier to view on the actions results page.